src/scss/lr7-classic-theme/_toolbars.scss

```diff
@@ -266,6 +262,65 @@
 	}
 }
 
+// Management Toolbar Skin
+
+.management-toolbar-default {
+	background-color: $toolbar-default-bg;
+	border-bottom: 1px solid $toolbar-bottom-border;
+	padding: 0 24px;
+
+	.btn {
+		background-color: transparent;
+		border-width: 0;
+		padding: 0;
+	}
+
+	.dropdown-menu {
+		margin-top: $toolbar-dropdown-margin-top;
+	}
+
+	.toolbar-group, .toolbar-group-right, .toolbar-group-expand {
+		height: $management-toolbar-height;
+
+		a, .btn, .btn-primary {
```

Comment:
Not sure why you’re only styling btn-primary here, what if someone uses btn-success instead?